src/components/Input/index.jsx

```diff
@@ -48,7 +50,7 @@ const Input = ({ type, value, onChange, placeholder }) => {
   return (
     <div className="input">
       <input
-        type={type}
+        type={type === "repeat-password" ? "password" : type}
```

Comment:
I don't think this is required. Let's sync on this

Reply:
Its used to check if the type is password or confirm password, so that it can make it dots, and not in the physical character

Reply:
Whether `password` or `confirmPassword`, the input value should be hidden when the user types.